.gitpod.Dockerfile

```diff
@@ -2,16 +2,5 @@ FROM gitpod/workspace-full:latest
 
 USER gitpod
 
-RUN npm i @learnpack/learnpack@2.1.20 -g && learnpack plugins:install @learnpack/python@1.0.0
-
-ENV VENV=coriseenv
-#RUN pip3 install pytest==4.4.2 mock pytest-testdox toml
-
-RUN python3 -m venv $VENV
-RUN ls .
-
-COPY requirements.txt .
-RUN . $VENV/bin/activate && pip install --upgrade pip==22.2.2
-RUN . $VENV/bin/activate && pip install -r requirements.txt
-
-CMD . $VENV/bin/activate 
\ No newline at end of file
+RUN pip3 install -U pip
```

Comment:
I would prefer to use virtual environments if we are doing "professional" Python.

Reply:
@mattharrison should it be activated at the vm start automatically or the student should do it? 

If the student does it then we can skip installing anything in the container altogether.